packages/search-ui-site-search-connector/src/responseAdapter.js

```diff
@@ -0,0 +1,17 @@
+import { getFacets, getResults } from "./responseAdapters";
+
+export default function adaptResponse(response, documentType) {
+  const results = getResults(response.records, documentType);
+  const totalPages = response.info[documentType].num_pages;
+  const totalResults = response.info[documentType].total_result_count;
+  const requestId = "";
+  const facets = getFacets(response.info[documentType]);
+
+  return {
+    results,
+    totalPages,
+    totalResults,
+    requestId,
+    ...(Object.keys(facets).length > 0 && { facets })
```

Comment:
I _think_ if facets has no keys/is empty then it won't spread anything, so the check for `Object.keys(facets).length` isn't necessary

Reply:
Confirmed. So that check is to avoid ending up with:
```
{
  facets: {}
}
```

When it's there, if there is an empty facets object, I simply get:
```
{
}
```

Which is what I want

Reply:
I see now that the goal was to include facets as a property under the object only when facets itself had properties, which this code does accomplish!